samples/peripheral/radio_test/README.rst

```diff
@@ -137,28 +138,26 @@ Building and running
 .. include:: /includes/build_and_run.txt
 
 .. note::
-   On the |nRF5340DKnoref|, the Radio test sample is a standalone network sample that does not require any counterpart application sample.
-   Program the application core to boot up the network core.
-   Use any sample for this, for example :ref:`nrf5340_empty_app_core`.
-   The :ref:`nrf5340_empty_app_core` is built and programmed automatically by default.
-   If you want to program another sample for the application core, unset the :kconfig:option:`CONFIG_NCS_SAMPLE_EMPTY_APP_CORE_CHILD_IMAGE` option.
+   On the nRF5340 development kit, the Radio Test sample requires :ref:`nrf5340_remote_shell` sample on the application core.
+   The :ref:`nrf5340_remote_shell` is built and programmed automatically by default.
+   If you want to program your custom solution for the application core, unset the :kconfig:option:`CONFIG_NCS_SAMPLE_REMOTE_SHELL_CHILD_IMAGE` option.
```

Comment:
```suggestion
   On the nRF5340 development kit, the Radio Test sample requires the :ref:`nrf5340_remote_shell` sample on the application core.
   The Remote IPC shell sample is built and programmed automatically by default.
   If you want to program your custom solution for the application core, unset the :kconfig:option:`CONFIG_NCS_SAMPLE_REMOTE_SHELL_CHILD_IMAGE` Kconfig option.
```
Same thing here. We may not want to link to the same thing twice in a row. Therefore, dumped the other link.